railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%%= locale %>">
```

Comment:
It's undefined in a view. You have to do `I18n.default_locale`. I think I'd still prefer `locale`, but `I18n.default_locale ` is hardly a bad solution.